docs/docsite/rst/reference_appendices/glossary.rst

```diff
@@ -396,6 +402,13 @@ when a term comes up on the mailing list.
         Ansible will never feel like assembly programming).  There are an
         infinite number of variable names you can use for registration.
 
+    Remote node
```

Comment:
We typically don't call this a `remote` node, but a `target` or `managed` node.  Since the node can actually be local.

Only modules execute on a target node.

Reply:
Yup, realised that after I progressed. I've asked Nitz to raise this at the architects meeting and confirm terminology, then I'll update.

Reply:
We did talk about this yesterday, and I think we decided on effectively:

* Control Node
* Target Node
* Some 3rd term to describe networking and cloud targets, since Target Node means the node where module code runs.  However with networking the module is running in a different place than the "resource" that is being managed.  This couldn't really been decided on.  We mentioned "automation target" or "target resource".

Note that in the product/platform we often call "target node" the "managed node", but managed carries extra connotations that don't universally apply.